### ISSUE TITLE
"Cannot bind parameter 'Option'" When running workflow

### ISSUE THREAD

**Issue by staffanhedstrom** [SanjulaGanepola/github-local-actions#78]
Hello!

This extension looked intriguing but I cannot get it to run. Act and Docker engine both show green.
![image](https://github.com/user-attachments/assets/a22d2526-9061-4683-86e7-b9db934fd048)

But any workflow I try to run produces this error:

```
 *  Executing task: set -o pipefail; act --workflows ".github/workflows/myworkflow.yml" --job "tests" --secret-file "" --var-file "" --input-file "" --eventpath "" 2>&1 | tee "c:\Users\myuser\AppData\Roaming\Code\User\globalStorage\sanjulaganepola.github-local-actions\MyTest #1 - 20241125_145859.log" 


Set-Variable : Cannot bind parameter 'Option'. Cannot convert value "pipefail" to type "System.Management.Automation.ScopedItemOptions". Error: "Unable to match the identifier name pipefail to a valid enumer
ator name. Specify one of the following enumerator names and try again:
None, ReadOnly, Constant, Private, AllScope, Unspecified"
At line:1 char:8
+ set -o pipefail; act --workflows .github/workflows/development_landei ...
+        ~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Set-Variable], ParameterBindingException
    + FullyQualifiedErrorId : CannotConvertArgumentNoMessage,Microsoft.PowerShell.Commands.SetVariableCommand
 
Tee-Object : A positional parameter cannot be found that accepts argument 'and'.
At line:1 char:159
+ ... ath  2>&1 | tee c:\Users\myuser\AppData\Roaming\Code\User\globa ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Tee-Object], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,Microsoft.PowerShell.Commands.TeeObjectCommand
 

 *  The terminal process "C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -Command set -o pipefail; act --workflows ".github/workflows/myworkflow.yml" --job "tests" --secret-file "" --var-file "" --input-file "" --eventpath "" 2>&1 | tee "c:\Users\myuser\AppData\Roaming\Code\User\globalStorage\sanjulaganepola.github-local-actions\MyTest #1 - 20241125_145859.log"" terminated with exit code: 1. 
 ```

VS Code is run as admin.

**Comment by SanjulaGanepola** [comment]
Hello @staffanhedstrom. Thank you for sharing this issue. I have a PR that I am testing and will merge soon. Will let you know when it is released.

**Comment by SanjulaGanepola** [comment]
@staffanhedstrom I just published a `1.1.2` release that contains the fix. Please let me know if you encounter the issue still.
